### ISSUE TITLE
Bottom step preview does not change steps in play mode

### ISSUE THREAD

**Issue by fominykh** [WEKIT-ECS/MIRAGE-XR#1374]
Currently:

related to #1376 

This is how the panel looks when the user is currently on step 1: step 1 is displayed + step 1 has a violet frame indicating that the step is active. ![image](https://user-images.githubusercontent.com/10297277/235447899-fb60ee73-02c3-4954-a227-4caa28bec515.png)

And this is how the panel looks when the user is currently on step 2 (after clocking the Next step button): step 1 is still displayed, but without the violet frame.
![image](https://user-images.githubusercontent.com/10297277/235447933-7dd5f15f-33b6-4b82-971d-9f9c3f81b2b6.png)

Should be:

This bottom panel should switch between steps and show the current step in the play mode.

**Comment by Alvareon** [comment]
Done:  https://github.com/WEKIT-ECS/MIRAGE-XR/pull/1402

![bottom_step_preview](https://user-images.githubusercontent.com/34060044/236940254-ff7a111a-731b-452a-aa57-5ae1d53c3c57.gif)
